docs/client.rst

```diff
@@ -0,0 +1,137 @@
+The SecureDrop Client Application
+=================================
+
+As part of the ongoing work to make a user (journalist?) friendly workstation
+for SecureDrop we have created a native client application to be run within the
+Qubes operating system. It helps journalists with the most common activities
+associated with using SecureDrop in a user friendly manner.
+
+Currently the client is alpha quality although work is ongoing in terms of
+improving features and the user interface.
+
+The source code, and related issues are `hosted on GitHub <https://github.com/freedomofpress/securedrop-client>`_.
+
+Developer Setup
+---------------
+
+Clone the repository::
+
+    git clone git@github.com:freedomofpress/securedrop-client.git
+
+Create and enable a new Python3 virtualenv::
+
+    python3 -m venv my_virtualenv
+    source my_virtualenv/bin/activate
+
+Set up the Python dependencies::
```

Comment:
This duplicates the steps that are in the client `README`. I think it would be better to say "follow the steps in the readme to init the dev env".

Reply:
I agree with the DRY sentiments here. That said I think we should keep all the "getting started" type developer docs in one place (i.e. docs.securedrop.org) - so we should migrate `README.md` content here and just provide a pointer to this page of documentation in the docs (providing a pointer to very detailed docs is what we do for the securedrop server readme, which I think is pretty nice)